packages/analytics/analytics-chart/src/components/chart-types/DoughnutChart.vue

```diff
@@ -58,14 +82,25 @@ const props = defineProps({
     required: false,
     default: null,
   },
+  chartTitle: {
+    type: String,
+    required: false,
+    default: null,
+  },
   fill: {
     type: Boolean,
     required: false,
     default: false,
   },
+  isSimple: {
+    type: Boolean,
+    required: false,
+    default: false,
+  },
```

Comment:
Do we want to have this, prop, that we need to have a test for, or should we make a separate, internal component called `GuageChart` for simple charts. 

Things like tooltips aren't necessary for simple charts, and the logic around them right so it would be simpler to have a smaller component that doesn't even have that logic in it.

Reply:
I'm torn on this. At the moment, the two chart types have enough props in common that it's not worth duplicating the template code. Let's chat after standup.